### PR TITLE
ATL-970: notification to last longer

### PR DESCRIPTION
**Pull request by fstasi**
With this PR I set the timeout of the following notification to 3 seconds (like other notifications in the IDE)
- burn bootloader
- verify sketch
- upload sketch

### CODE REVIEW

**Comment by ubidefeo** [comment]
@fstasi 
the build fails, I'll test this tomorrow

**Comment by per1234** [comment]
@ubidefeo I reran the build and it passed this time so it's all ready for testing. The failure was only caused by a temporary external network outage; nothing related to the code here.

**Review by kittaakos — CHANGES_REQUESTED** [review]
The code change looks good to me, but the base branch is incorrect for this PR. This PR should not contain the keymap + preferences changes. Please fix it. Thank you!

**Review by kittaakos — APPROVED** [review]
Thank you! The code looks to me. 👍

Please merge once you have the final approval.

**Review by rsora — APPROVED** [review]
Tests on Win are 👍

**Pull request merged**
